python/dgl/transform/functional.py

```diff
@@ -2993,10 +2993,11 @@ def reorder_graph(g, node_permute_algo=None, edge_permute_algo='src',
     ----------
     g : DGLGraph
         The homogeneous graph.
-    node_permute_algo: str, optional
+    node_permute_algo: None or str, optional
```

Comment:
It's ok not to mention `None` for simplicity. Sphinx will show the default value in the API signature.